book/pipelines.md

```diff
@@ -118,7 +118,7 @@ With most filters, the second version would refer to the entire `file` record (w
 
 ### Pipeline input in custom command definitions and scripts
 
-See: [Custom Commands -> Pipeline Input](http://localhost:8080/book/custom_commands.html#pipeline-input)
+See: [Custom Commands -> Pipeline Input](custom_commands.html#pipeline-input)
```

Comment:
Isn't book still missing here?

Reply:
`/book/` isn't necessary here since `custom_commands...` is relative to it already.

Reply:
ok, it's better than localhost anyhow